variable_info.txt

```diff
@@ -0,0 +1,11 @@
+name      - the name of the galaxy; 
```

Comment:
This is from the previous assignment. Be sure to delete unneccessary files before the deadline ;)